docs/css/style.css

```diff
@@ -0,0 +1,81 @@
+/* Docs Stylesheet */
+
+body {
+  background-color: var(--light-color);
+  color: var(--dark-color);
+}
+
+.brand {
+  color: var(--primary-color);
+}
+
+.docs-container {
+  display: flex;
+  justify-content: space-between;
+  height: auto;
+}
+.docs-container > .sidebar-left {
+  flex: 1 0 auto;
+  /* outline: 1px solid black; */
+  /* overflow: auto; */
+  width: 12rem;
+  /* height: 100vh; */
+}
```

Comment:
Nice UI !! Just try removing these extra code comments if not of use

Reply:
Thanks for pointing it out @simran61, Fixed it now.